tools/deploy.py

```diff
@@ -53,17 +49,36 @@ def deploy_pew_pew_control_module(containerPath):
 def deploy_py3status_module():
     configHomePath = Path(os.getenv("XDG_CONFIG_HOME", "~")).expanduser()
     candidates = [
-        configHomePath / ".config/py3status/modules",
-        configHomePath / ".config/i3status/py3status",
-        configHomePath / ".config/i3/py3status",
-        configHomePath / ".i3/py3status",
+        configHomePath / ".config" / "py3status" / "modules",
+        configHomePath / ".config" / "i3status" / "py3status",
+        configHomePath / ".config" / "i3" / "py3status",
+        configHomePath / ".i3" / "py3status",
     ]
     path = first_existing(candidates)
     if path is not None:
         log.info(f"deploy py3status module {PY3STATUS_MODULE_PATH} to {path}")
         shutil.copy(PY3STATUS_MODULE_PATH, path)
     else:
-        raise Exception("Could not find a home for pewpew")
+        raise Exception("Could not find a home for {}".format(MODULE_NAME))
+
+
+def warn_if_user_not_in_expected_groups():
+    user = getpass.getuser()
+    groups = [g.gr_name for g in grp.getgrall() if user in g.gr_mem]
+    expected_groups = ['input', 'dialout']
+    for exp_group in expected_groups:
+        if exp_group not in groups:
+            print("Warning: user '{}' is not in expected group: '{}'".format(user, exp_group))
```

Comment:
Use log.warning instead of print?

Reply:
On second thought: this should be either an info as we 're only guessing that there could be a problem. What would be even better than checking for groups would be for actually checking, if we can do what we need to be able to do ... tht will definitely a bit trickier though.

Reply:
Changed to logger.info.